firstdata.php

```diff
@@ -19,7 +19,7 @@ public function __construct()
 	{
 		$this->name = 'firstdata';
 		$this->tab = 'payments_gateways';
-		$this->version = '1.2.5';
+		$this->version = '1.2.6';

```

Comment:
The current version of the module is 1.2.7, so I guess that your fork is not up-to-date ?